lib/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ForwardRefRenderFunction } from "react";
+import { textAreaStyle } from "./text-area.css";
+import { TextAreaProps } from "./text-area.types";
+
+const TextAreaComponent: ForwardRefRenderFunction<HTMLTextAreaElement, TextAreaProps> = ({ 
+    id,
+    name,
+    autoComplete = 'off',
+    autoCorrect = 'off',
+    autoFocus,
+    cols = 35,
+    rows = 5,
+    disabled,
+    form,
+    maxLength,
+    minLength,
+    placeHolder = 'Placeholder',
```

Comment:
Let default placeholder be blank, helps keep behaviour close to normal textArea

Reply:
okay !